tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -115,6 +152,39 @@ def tap_switch_source(self):
         Wait(self.marionette).until(lambda m: not current_camera_mode == self.camera_mode)
         self.wait_for_capture_ready()
 
+    def tap_switch_to_front_camera(self):
```

Comment:
the function definitions don't seem to be indented properly.  Perhaps running pep8 tool would help?